src/Event/PartialEvent.php

```diff
@@ -94,6 +96,13 @@ public function addParticipation(BaseEventParticipation $participation)
         parent::addParticipation($participation);
     }
 
+    public function setParticipations(ArrayCollection $participations)
+    {
```

Comment:
control what is in the collection, as you could have a collection of `string` for example.